tools/scripts/build/build-curriculum.ts

```diff
@@ -11,13 +14,16 @@ const globalConfigPath = path.resolve(__dirname, '../../../config');
 // We are defaulting to English because the ids for the challenges are same
 // across all languages.
 getChallengesForLang('english')
```

Comment:
```suggestion
void getChallengesForLang('english')
```
This lets the linter know we're deliberately not catching or awaiting.